src/steps/video-setup/preview.tsx

```diff
@@ -145,3 +149,51 @@ const PreviewVideo: React.FC<{ input: Input }> = ({ input }) => {
     </div>
   );
 };
+
+export const DisplayAudioInfo: React.FC<{ stream: MediaStream }> = ({ stream }) => {
+  const hasAudio = stream.getAudioTracks().length;
+
+  return (
+    <div css={{
+      position: "absolute",
+      top: 8,
+      right: 8,
+      whiteSpace: "pre-line",
+    }}>
+      <WithTooltip
+        placement="bottom"
+        tooltip={<Trans i18nKey={
+          `steps.video.${hasAudio ? "display-audio-shared" : "display-audio-not-shared"}`
+        }>
+          <strong>Note:</strong> Explanation.
+        </Trans>}
```

Comment:
```suggestion
        tooltip={
            <Trans i18nKey={
              `steps.video.${hasAudio ? "display-audio-shared" : "display-audio-not-shared"}`
            }>
              <strong>Note:</strong> Explanation.
            </Trans>
        }
```

That fits the existing code style better.